pkg/sql/sqlbase/type_desc_test.go

```diff
@@ -296,59 +309,65 @@ func TestValidateTypeDesc(t *testing.T) {
 						PhysicalRepresentation: []byte{2},
 					},
 				},
+				Privileges: defaultPrivileges,
 			},
 		},
 		{
 			`ALIAS type desc has nil alias type`,
 			descpb.TypeDescriptor{
-				Name:     "t",
-				ID:       1,
-				ParentID: 1,
-				Kind:     descpb.TypeDescriptor_ALIAS,
+				Name:       "t",
+				ID:         typeDescID,
+				ParentID:   1,
+				Kind:       descpb.TypeDescriptor_ALIAS,
+				Privileges: defaultPrivileges,
 			},
 		},
 		{
 			`parentID 500 does not exist`,
 			descpb.TypeDescriptor{
-				Name:     "t",
-				ID:       1,
-				ParentID: 500,
-				Kind:     descpb.TypeDescriptor_ALIAS,
-				Alias:    types.Int,
+				Name:       "t",
+				ID:         typeDescID,
+				ParentID:   500,
+				Kind:       descpb.TypeDescriptor_ALIAS,
+				Alias:      types.Int,
+				Privileges: defaultPrivileges,
 			},
 		},
 		{
 			`parentSchemaID 500 does not exist`,
 			descpb.TypeDescriptor{
 				Name:           "t",
-				ID:             1,
+				ID:             typeDescID,
 				ParentID:       100,
 				ParentSchemaID: 500,
 				Kind:           descpb.TypeDescriptor_ALIAS,
 				Alias:          types.Int,
+				Privileges:     defaultPrivileges,
 			},
 		},
 		{
 			"arrayTypeID 500 does not exist",
 			descpb.TypeDescriptor{
 				Name:           "t",
-				ID:             1,
+				ID:             typeDescID,
 				ParentID:       100,
 				ParentSchemaID: 101,
 				Kind:           descpb.TypeDescriptor_ENUM,
 				ArrayTypeID:    500,
+				Privileges:     defaultPrivileges,
 			},
 		},
 		{
 			"referencing descriptor 500 does not exist",
 			descpb.TypeDescriptor{
 				Name:                     "t",
-				ID:                       1,
+				ID:                       typeDescID,
 				ParentID:                 100,
 				ParentSchemaID:           101,
 				Kind:                     descpb.TypeDescriptor_ENUM,
 				ArrayTypeID:              102,
 				ReferencingDescriptorIDs: []descpb.ID{500},
+				Privileges:               defaultPrivileges,
```

Comment:
should add a case where validation fails

Reply:
Done.